Source/WebKit/UIProcess/WebBackForwardList.cpp

```diff
@@ -456,24 +456,30 @@ void WebBackForwardList::restoreFromState(BackForwardListState backForwardListSt
 
     // FIXME: Enable restoring resourceDirectoryURL.
     m_entries = WTF::map(WTFMove(backForwardListState.items), [this](auto&& state) {
-        state.identifier = BackForwardItemIdentifier::generate();
+        Function<void(FrameState&)> setIdentifier;
+        setIdentifier = [&setIdentifier](FrameState& frameState) {
```

Comment:
This method of making a recursive lambda isn't really needed here because nothing else is captured.  A recursive function would do just as well and be a little nicer in my opinion.  But it wouldn't be as well scoped.